tests/test_diffeq/test_perturbedsolvers/test_perturbedstepsolver.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+import pytest
+import pytest_cases
+from scipy.integrate._ivp import base, rk
+
+from probnum import diffeq, randvars
+from probnum.diffeq.perturbedsolvers import _perturbation_functions
+
+
+@pytest_cases.fixture
+@pytest_cases.parametrize_with_cases(
+    "testsolver, perturbedsolver", cases="test_perturbed_cases"
+)
+def solvers(testsolver, perturbedsolver):
+    return testsolver, perturbedsolver
+
+
+@pytest.fixture
+def start_point():
+    return 0.0
+
+
+@pytest.fixture
+def stop_point():
+    return 0.1
+
+
+@pytest.fixture
+def y():
+    return randvars.Constant(0.1)
+
+
+@pytest.fixture
+def dense_output():
+    return [base.DenseOutput(0, 1)]
+
+
+@pytest.fixture
+def times():
+    return [0, 1]
+
+
+@pytest.fixture
+def steprule():
+    return diffeq.ConstantSteps(0.1)
+
+
+@pytest.fixture
+def list_of_randvars():
+    return list([randvars.Constant(1)])
+
+
+@pytest.fixture
+def deterministicsolver():
+    y0 = np.array([0.0, 1.0, 1.05])
+    ode = diffeq.lorenz([0.0, 1.0], y0)
+    testsolver = diffeq.WrappedScipyRungeKutta(rk.RK45(ode.rhs, ode.t0, y0, ode.tmax))
+    return diffeq.PerturbedStepSolver(
+        testsolver,
+        noise_scale=1,
+        perturb_function=_perturbation_functions.perturb_uniform,
+        random_state=123,
+    )
```

Comment:
is this deterministic? The name suggests that, but it calls PerturbedStepSolver.